src/components/Stories.js

```diff
@@ -186,6 +180,11 @@ const Stories = ({ authorId, followerId }) => {
 
   return (
     <div>
+      <ProductSelect
```

Comment:
Lets make this `<ProductList products={products} />` and maintain state in `ProductList` component.